src/index.html

```diff
@@ -4,9 +4,377 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#" class="logo-link">
+        <img src="./images/logo.png" alt="logoMoyo">
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="index.html" class="nav__link is-active">Apple</a>
+          </li>
+          <li class="nav__item">
+            <a href="index.html" class="nav__link">Samsung</a>
+          </li>
+          <li class="nav__item">
+            <a href="index.html" class="nav__link">Smartphones</a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="index.html"
+              class="nav__link"
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="index.html" class="nav__link">Gadgets</a>
+          </li>
+          <li class="nav__item">
+            <a href="index.html" class="nav__link">Tablets</a>
+          </li>
+          <li class="nav__item">
+            <a href="index.html" class="nav__link">Photo</a>
+          </li>
+          <li class="nav__item">
+            <a href="index.html" class="nav__link">Video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="page">
+      <div class="catalog page__catalog-out">
+        <div class="card catalog__card-out" data-qa="card">
+          <img
+            class="card__img"
+            src="./images/imac.jpeg"
+            alt="Apple Monoblock"
+          >
+
+          <h1 class="card__title">
```

Comment:
h1 can be only one at page, change to h2

Reply:
interesting, something new) 
thanks